docs/external_system.md

```diff
@@ -248,7 +248,7 @@ steps needed to configure SAT to use externally-accessible API endpoints exposed
    username = "user"
    ```
 
-## SAT Authentication
+## Authenticate SAT Commands
```

Comment:
I don't think I can comment on this directly (since the relevant lines are too far from the contents of this PR) but I just wanted to point out a couple issues that have been uncovered now that `external_system.md` is a separate document.

"Authenticate SAT Commands" and "Generate S3 Credentials" have been pasted in verbatim in this document but they would look slightly different if running them on an external system. Really this was an issue before, just less obvious because the instructions were not directly in the same document.

The first issue is to do with command prompts. Not a huge issue, but they all start with "ncn-m001" even though in this document you're setting things up on an external system. It would be nice to change them all to `$` like we are doing earlier in the instructions.

Second, step 4 of "Generate S3 Credentials" instructs you to use `scp` to copy the config files too all the different nodes of the system. This is a step that might make sense in its original context, but doesn't really make sense here.

It is probably going to be challenging to maintain two slightly divergent sets of instructions, one for internal system and one for external systems. To at least partially address that I might suggest that we try to trim down the versions found in `external_system.md`. For example, we could remove the "Description of SAT Command Authentication Types" table, or better yet put it into `introduction.md` where it's always belonged :)

We could probably tackle this in a follow-up Jira, since it seems a little open-ended right now.

Reply:
Thanks for pointing this out! I agree some changes need to be made, and I created [STP-3522](https://jira-pro.its.hpecorp.net:8443/browse/STP-3522) for this work in a later PR.